src/gmm_utils.erl

```diff
@@ -28,15 +28,19 @@
     create_edge_id/2,
     split_edge_id/1,
 
-    convert_microseconds_to_iso_8601/1,
-    parse_boolean/1,
+    list_other_zones/0,
+    batch_size/0,
     convert_microseconds_to_iso_8601/1,
     permissions_and/2,
     permissions_or/2
 ]).
 
 -include("records.hrl").
 
+-type permissions() :: <<_:5*8>>.
+-export_type([
+    permissions/0
+]).
```

Comment:
Nie jestem pewien co do eksportowania tego typu. Bardziej dałbym to do jakiegoś headera.

Reply:
Zrobić header.hrl bezpośrednio w src/ ?

Reply:
Możesz dodać do tego headera który już mamy, ewentualnie zmienić mu nazwę ale nie wiem czy jest sens